src/main/java/com/unblu/uneedswork/service/GitLabService.java

```diff
@@ -77,8 +77,8 @@ public UNeedsWorkResult handleEvent(NoteEventSimple event) {
 				}
 
 			} else {
-				Log.infof("GitlabEvent: '%s' | Skip event (not relevant) for Project: '%d', NoteId: '%d', NoteAuthorId: '%d', UserName: '%s', MrIid: '%d'",
-						gitlabEventUUID, projectId, noteId, noteAuthorId, userName, mrIid);
+				Log.infof("GitlabEvent: '%s' | Skip event (not relevant) for Project: '%d', NoteId: '%d', NoteAuthorId: '%d', UserName: '%s', MrIid: '%d', NoteType: '%s'",
+						gitlabEventUUID, projectId, noteId, noteAuthorId, userName, mrIid, event.getNoteType());
 			}
 		} else {
 			Log.infof("GitlabEvent: '%s' | Skip event (no MR) for Project: '%d', NoteId: '%d', NoteAuthorId: '%d', UserName: '%s', MrIid: '%d'",
```

Comment:
If the MR is not set, it means the comment was posted somewhere else (issue, commit, …). So I don't think we need to log the type.

I added it in the other place since if in a future version of GitLab, they set a type when a review is posted, then our tool will skip the comment and the log allow to understand why it was filtered.